docs/getting-started-guides/minikube.md

```diff
@@ -0,0 +1,76 @@
+---
+---
+
+* TOC
+{:toc}
+
+### Requirements
+
+Minikube requires that VT-x/AMD-v virtualization is enabled in BIOS.
```

Comment:
Also, add a brief introduction of the purporse.  `Minikube starts a single node kubernetes cluster locally for purposes of development and testing.`